src/frontend/src/optimizer/plan_node/stream_expand.rs

```diff
@@ -42,7 +41,7 @@ impl StreamExpand {
             Distribution::Broadcast => unreachable!(),
         };
 
-        let mut watermark_columns = FixedBitSet::with_capacity(schema.len());
+        let mut watermark_columns = FixedBitSet::with_capacity(logical.output_len());
```

Comment:
a bug, which did not be founded because we do not use expand in streaming for a long time